docs/api.rst

```diff
@@ -0,0 +1,19 @@
+PyLmod API Docs
+---------------
+
+For convenient reference in development, here are the PyLmod
+API docs.
+
+Library
+=======
+
+API documentation
+
+.. automodule:: pylmod
```

Comment:
I agree.  Loading each class specifically provides superior control over what appears in the docs.